funnel/forms/project.py

```diff
@@ -39,22 +39,26 @@ class ProjectForm(forms.Form):
         validators=[forms.validators.DataRequired()],
         filters=[forms.filters.strip()],
     )
-    location = forms.StringField(
-        __("Location"),
-        validators=[forms.validators.DataRequired(), forms.validators.Length(max=50)],
-        filters=[forms.filters.strip()],
-        description=__("Eg. Bangalore, Mumbai, Pune"),
-    )
     tagline = forms.StringField(
         __("Tagline"),
         validators=[forms.validators.DataRequired(), forms.validators.Length(max=250)],
         filters=[forms.filters.strip()],
         description=__("One line description of the project"),
     )
-    description = forms.MarkdownField(
-        __("Project description"),
-        validators=[forms.validators.DataRequired()],
-        description=__("Landing page contents"),
+    location = forms.StringField(
+        __("Location"),
+        description=__(
+            '“Online”, “Bangalore”, “Chennai”, “Pune”, etc (without quotes)'
```

Comment:
I'd update this to be friendlier for the times:

`“Online” if this is online-only, else the city or region (without quotes)`